my-site.rb

```diff
@@ -2,7 +2,15 @@
 
 class MySite < Sinatra::Base
 
-  get "/" do
```

Comment:
It's still good to have the default "root route" in your page so that a user will be redirected to _something_ if they don't navigate to an appropriate URL